### PR TITLE
DOCSP-41978: Count documents

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41978
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41978-count/read/count/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/count.txt [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+.. _php-accurate-count:
+
+Retrieve an Accurate Count
+--------------------------
+
+Use the ``MongoDB\Collection::countDocuments()`` method to count the number of documents
+in a collection. To count the number of documents that match specific search criteria,
+pass a query filter to the ``countDocuments()`` method.
+
+To learn more about specifying a query, see the :ref:`php-specify-query` guide.
```
Comment (norareidy) [inline source/read/count.txt:55]: Note: link broken until #103 is merged

File: source/read/count.txt [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+      Number of companies with 50 employees: 100
+
+.. _php-estimated-count:
+
+Retrieve an Estimated Count
+---------------------------
+
+You can retrieve an estimate of the number of documents in a collection by calling
+the ``estimatedDocumentCount()`` method. The method estimates the amount of
```
Comment (rustagir) [inline source/read/count.txt:167]: S: in other method references, you include the class that the method is part of

File: source/read/count.txt [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+         :manual:`Read Preference </core/read-preference/>` in the Server manual.
+       | **Type**: ``MongoDB\Driver\ReadPreference``
+
+   * - ``session``
+     - | The client session to associate with the operation.
+       | **Type**: ``MongoDB\Driver\Session``
+
+The following example uses the ``estimatedDocumentCount()`` method to return an
+estimate of the number of documents in the collection and instructs the operation
+to run for a maximum of ``1000`` milliseconds:
```
Comment (rustagir) [inline source/read/count.txt:225]:
```suggestion
The following example uses the ``estimatedDocumentCount()`` method to return an
estimate of the number of documents in the collection and sets a timeout of
``1000`` milliseconds on the operation:
```

**Review by rustagir — CHANGES_REQUESTED** [review]
a few small improvements!

**Review by rustagir — APPROVED** [review]

**Review by GromNaN — APPROVED** [review]

**Pull request merged**